core-tests/src/test/java/org/mule/runtime/core/routing/filters/ExpressionFilterTestCase.java

```diff
@@ -137,15 +136,14 @@ public void testRegexFilterWithAngleBrackets() {
 
   @Test
   public void testExceptionTypeFilter() {
-    ExpressionFilter filter = new ExpressionFilter("exception is java.lang.Exception");
+    ExpressionFilter filter = new ExpressionFilter("exception.getCause() is java.lang.Exception");
```

Comment:
This is caused by wrapping the exception in a MessagingException to support the "causedBy" and I think it should go away once we can get rid of all those uses. Should I add the "causedBy" stuff to the migration and create a ticket to get rid of it?